applications/OptimizationApplication/python_scripts/output/container_variable_data_holder_output_decorator.py

```diff
@@ -0,0 +1,228 @@
+from abc import ABC
+from abc import abstractmethod
```

Comment:
```suggestion
from abc import ABC, abstractmethod
```
same for other multiple imports